From a83a68b85ca41e9b4d4b99d087697d891b9290a0 Mon Sep 17 00:00:00 2001
From: Andrey Utkin <andrey_utkin@fastmail.com>
Date: Fri, 7 Sep 2018 01:23:03 +0100
Subject: [PATCH 3/3] Allow uid to be in event print format

---
 khal/khalendar/event.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/khal/khalendar/event.py b/khal/khalendar/event.py
index 1eaf37e..eb7ebc4 100644
--- a/khal/khalendar/event.py
+++ b/khal/khalendar/event.py
@@ -581,6 +581,7 @@ class Event(object):
 
         attributes['status'] = self.status
         attributes['cancelled'] = 'CANCELLED ' if self.status == 'CANCELLED' else ''
+        attributes['uid'] = self.uid
         return format_string.format(**dict(attributes)) + attributes["reset"]
 
     def duplicate(self):
-- 
2.16.4

